frontend/i18n-scripts/memsource-upload.sh

```diff
@@ -0,0 +1,35 @@
+#!/usr/bin/env bash
+
+set -exuo pipefail
+
+source ./i18n-scripts/languages.sh
+
+while getopts v:s: flag
+do
+  case "${flag}" in
+      v) VERSION=${OPTARG};;
+      s) SPRINT=${OPTARG};;
+      *) echo "usage: $0 [-v] [-s]" >&2
+      exit 1;;
+  esac
+done
+
+echo "Creating project with title \"[OCP $VERSION] UI Localization - Sprint $SPRINT\""
+
+PROJECT_INFO=$(memsource project create --name "[OCP $VERSION] UI Localization - Sprint $SPRINT" --template-id 169304 -f json)
```

Comment:
We should include the Git branch in here somewhere so that we can translate z-stream releases.